dagmc_stats.py

```diff
@@ -300,3 +300,178 @@ def get_coarseness(my_core, meshset, entity_ranges, geom_dim):
         coarseness.append(len(surf_area)/sum(surf_area))
 
     return coarseness
+
+
+def get_beta_angles(my_core, vert, verts, vert_dic):
+    """
+    Gets the beta angles of corresponding vertices pairs
+
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    vert : entity handle of the central vertex
+    verts : entity handles of the adjacent vertices of vert
+    vert_dic : a dictionary storing common vertex:beta angles
+    """
+    
+    verts.remove(vert)
+    side_lengths = []
+    lst1 = sorted([vert, verts[0]])
+    lst2 = sorted([vert, verts[1]])
```

Comment:
```suggestion
    lst1 = str(sorted([vert, verts[0]]))
    lst2 = str(sorted([vert, verts[1]]))
```
You only use these variables as a dictionary key, so they can be converted to strings right away. See comments below for corresponding code changes.